src/components/Tags/Tag.tsx

```diff
@@ -0,0 +1,26 @@
+import {FunctionComponent} from "react";
+import {Label, LabelProps} from "@patternfly/react-core";
+import {APILabel} from "@apidocs/common";
+import assertNever from "assert-never";
+
+interface TagProps {
+    value: Readonly<APILabel>;
+}
+
+const colorForLabelType = (type: APILabel['type']): LabelProps['color'] => {
+    switch (type) {
+        case "platform":
+            return 'green';
+        case "service":
+        case "use-case":
+            return 'blue';
+        default:
```

Comment:
@josejulio  Question: Is the service case missing a return statement? Or, it it getting inferred from "use-case"?

Reply:
It's using the same as service. I saw in the mocks that both were blue